docs/kafka/consumer-configuration-kafka/README.adoc

```diff
@@ -139,7 +139,7 @@ sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule require
 == Adding a consumer to a consumer group
 
 [role="_abstract"]
-Consumer groups are created by specifying a consumer group ID in the user's client application, and not from the CLI or web console.
+To create a consumer group you specify a consumer group ID in a client application. You do *not* create consumer groups in the {product-kafka} web console, or using the CLI.
```

Comment:
looks ok, but might be better changing "in a client application" to "in your consumer application"

Reply:
appplied